src/styles/block/header-style.scss

```diff
@@ -0,0 +1,87 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
+body {
+  padding: 0;
+  margin: 0 auto;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  height: 60px;
+  position: relative;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  text-align: center;
+}
+
+.nav__list {
+  display: flex;
+}
+
+.nav__item {
+  list-style-type: none;
+  display: inline-block;
```

Comment:
According to the Code Style Rules, `.nav__item` should use `margin-right` for spacing instead of `display: inline-block;` which is not needed here as the parent uses `display: flex;`.